internal/handlers/pg/msg_explain.go

```diff
@@ -84,7 +84,9 @@ func (h *Handler) MsgExplain(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 	var port int32
 	connInfo := conninfo.GetConnInfo(ctx)
 	if connInfo.PeerAddr != nil {
-		port = int32(connInfo.PeerAddr.(*net.TCPAddr).Port)
+		if tcpAddr, ok := connInfo.PeerAddr.(*net.TCPAddr); ok {
+			port = int32(tcpAddr.Port)
```

Comment:
Do I understand correctly that if we want to test it, we need to run tests without `-target-unix-socket`? 
That looks a bit tricky as in `Taskfile.yml` we don't set  `-target-unix-socket` only for Tigris, and this case will not be covered (because it's the one for `pg`).

Reply:
Yes, without `-target-unix-socket` uses TCP. With current Taskfile.yml setup we won't be testing port related thing on CI.